include/kernel/syscall.h

```diff
@@ -28,6 +28,7 @@ typedef enum {
 	SYSCALL_WRITE,
 	SYSCALL_LSEEK,
 	SYSCALL_CLOSE,
+  SYSCALL_FILE_DESCRIBE,
```

Comment:
How about `SYSCALL_OBJECT_TYPE` b/c this isn't necessarily about a file?

Might also be an opportune time to name all of the object operations `object_{open|read|write|etc}`

Reply:
Changed the enumeration, but to clarify:
   1. Should the function name also change?
   2. Is the additional change being requested is to cange every syscall that deals with file descriptors to obect_<call> syntax>

Reply:
1 . Yes, all the relevant functions should be called `object_type` so its clear how things work from top to bottom.

2. Yes, that is where I would like to go, however, since that is likely to be a high-conflict change, it's ok if you put that off for another PR.